binding/form_mapping.go

```diff
@@ -23,6 +24,15 @@ func mapForm(ptr interface{}, form map[string][]string) error {
 
 		structFieldKind := structField.Kind()
 		inputFieldName := typeField.Tag.Get("form")
+		inputFieldNameList := strings.Split(inputFieldName, ",")
+		inputFieldName = inputFieldNameList[0]
+		var defaultValue interface{}
```

Comment:
any need to cast it as `interface{}`? As I see below, you'll always use it as string and parse it to the specific type, right?

Reply:
Fixed, use `string`, thanks!